beamer/tests/health/conftest.py

```diff
@@ -0,0 +1,43 @@
+import pytest
+
+from beamer.health.check import Context
+from beamer.health.notify import NotificationState
+from beamer.tests.health.test_check import AGENT_ADDRESS
```

Comment:
`conftest.py` is a configuration file for `pytest` and as such runs fairly early in the testing session and other tests actually depend on `conftest.py` things, so we should not introduce a reverse dependency here.

My suggestion is to move `AGENT_ADDRESS` here and make it either a package- or a session-scoped fixture,
e.g.

```python
@pytest.fixture(scope="session")
def agent_address():
    return make_address()
```